code/index.html

```diff
@@ -1,22 +1,192 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
 
-    <section class="">
-      <!-- The other news  -->
+<head>
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans+Condensed:wght@300&display=swap" rel="stylesheet">
+  <meta charset="utf-8" />
+  <title>Project 2: The New PLANT Site</title>
+  <link rel="stylesheet" href="./style.css" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+
+<body>
+  <header>
+    <!-- Navbar: links to other plants pages? Inga länkar bara text -->
+    <div class="intro">
+      <nav>
+        <a href="#">Contact</a>
+        <a href="#">More inspo</a>
+        <a href="#">My Basket</a>
+      </nav>
+    </div>
+
+  </header>
+
+  <section class="big-news">
+    <img src="bladpårad.jpg" alt="Leaves in a row">
+
+    <div>
+      <h1> No green fingers? </h1>
+      <p>No problems! We got you covered!
+        We've selected the prettiest and the easiest plants to take care of. <br>
+        You'll the benefits of having a small jungle at home, without having to put
+        (almost) any effort into it!
+      </p>
+    </div>
+
+
+  </section>
+
+  <section class="plants">
+    <!-- Behöver något NEWS:igt! och kanske en stor bild?-->
+    <!-- se över denna text! och gör den smalare! -->
+    <h4> Here we have selected a few of the most beautiful and easy plants. <br>
```

Comment:
We should avoid using `br` tags as much as we can - you can always wrap some part of text inside this `h4` element with another element and force it to be in next line by `display: block`